### DISCUSSION TITLE
Sign message does not work using Coinbase Wallet on iOS Safari

### DISCUSSION THREAD

**Discussion by Rohitbaghel11** [wevm/wagmi#4157] [Q&A]
I'm having problem with Coinbase Wallet on iOS (did not tested for other devices, but I assume it's the same issue). Basically, I'm able to connect wallet using Coinbase, but when I try so Sign, I get error 'UnknownRpcError'.
error code 3000

I'm using wagmi useSignMessage and it works perfectly with Metamask, but only mobile Coinbase wallet app does not throw popup asking for Sign.

As well for connector I'm using RainbowConnect, and for Signing message - Wagmi package

**Comment by cristobalbahe** [comment]
Happens to me too. Metamask works perfectly but coinbase wallet does  not.

I am using wallet connect appkit to connect and useSignMessage from Wagmi.

**Comment by Ghadi8** [comment]
For it to work, you will have to disable the "Block Pop-ups" feature from your safari settings. Open the Settings app on your device, select "Safari" under "Apps", locate the "Block Pop-ups" feature and disable it. You should now be able to sign messages using your Coinbase wallet and smart wallet.
